Makefile

```diff
@@ -1,12 +1,22 @@
 all: test
 
-uninstall:
-	pip freeze | grep -v "^-e" | xargs pip uninstall -y
-
-install:
+install.tools:
 	pip install --upgrade pip twine wheel
+
+## Install openfisca-core for deployment and publishing.
+install.build: setup.py clean install.tools
+	@## This allows us to be sure tests are run against the packaged version
+	@## of openfisca-core, the same we put in the hands of users and reusers.
+	pip install --upgrade pip wheel
+	python $< bdist_wheel
```

Comment:
Using [`$<`](https://stackoverflow.com/questions/3220277/what-do-the-makefile-symbols-and-mean) costs making this list of instructions ununderstandable to anyone who doesn't know the `make` special shortcuts. What do we gain from using it instead of repeating `setup.py`? 😉